Update Dimension to budget by description (#36229)
DataDog/documentation: content/en/cloud_cost_management/planning/budgets.md

   - {{< ui >}}Start Date{{< /ui >}}: Enter a start date for the budget (this can be a past month). Budgets are set at the month level.
   - {{< ui >}}End Date{{< /ui >}}: Set an end date for the budget (can be in the future).
   - {{< ui >}}Provider(s){{< /ui >}}: Budget on any combination of AWS, Azure, Google Cloud, Oracle Cloud, or other SaaS (including Datadog or custom costs).

>    - {{< ui >}}Dimension to budget by{{< /ui >}}: Specify the dimension you want to track (for example, team, service, or environment). After selecting the dimension, define the specific values directly in the budget table. For example, to create budgets for the top four teams, select "team" as the dimension, then add those teams as rows in the table.

Reviewer: ```suggestion
   - {{< ui >}}Dimension to budget by{{< /ui >}}: Specify the dimension to track (such as, team, service, or environment). Then, define the specific values directly in the budget table. For example, to create budgets for the top four teams, select "team" as the dimension, and add the teams as rows in the table.
```
Edit suggestion for clarification. Lmk if this keeps the meaning.

   - {{< ui >}}Scope to Provider(s){{< /ui >}}: Budget on any combination of AWS, Azure, Google Cloud, Oracle Cloud, or other SaaS (including Datadog or custom costs).
   - {{< ui >}}Parent Level{{< /ui >}}: Select the parent-level tag.
   - {{< ui >}}Child Level{{< /ui >}}: Select child-level tag.

>    - {{< ui >}}Dimension to budget by{{< /ui >}}: Specify the dimension you want to track (for example, team, service, or environment). After selecting the dimension, define the specific values directly in the budget table. For example, to create budgets for the top four teams, select "team" as the dimension, then add those teams as rows in the table.

Reviewer: ```suggestion
   - {{< ui >}}Dimension to budget by{{< /ui >}}: Specify the dimension to track (such as, team, service, or environment). Then, define the specific values directly in the budget table. For example, to create budgets for the top four teams, select "team" as the dimension, and add the teams as rows in the table.
```
Edit suggestion for clarification. Lmk if this keeps the meaning.
